### ISSUE TITLE
main.py: error: the following arguments are required: task

### ISSUE THREAD

**Issue by 201814476285951** [xingyizhou/CenterNet#200]
usage: main.py [-h] [--dataset DATASET] [--exp_id EXP_ID] [--test]
               [--debug DEBUG] [--demo DEMO] [--load_model LOAD_MODEL]
               [--resume] [--gpus GPUS] [--num_workers NUM_WORKERS]
               [--not_cuda_benchmark] [--seed SEED] [--print_iter PRINT_ITER]
               [--hide_data_time] [--save_all] [--metric METRIC]
               [--vis_thresh VIS_THRESH] [--debugger_theme {white,black}]
               [--arch ARCH] [--head_conv HEAD_CONV] [--down_ratio DOWN_RATIO]
               [--input_res INPUT_RES] [--input_h INPUT_H] [--input_w INPUT_W]
               [--lr LR] [--lr_step LR_STEP] [--num_epochs NUM_EPOCHS]
               [--batch_size BATCH_SIZE]
               [--master_batch_size MASTER_BATCH_SIZE] [--num_iters NUM_ITERS]
               [--val_intervals VAL_INTERVALS] [--trainval] [--flip_test]
               [--test_scales TEST_SCALES] [--nms] [--K K]
               [--not_prefetch_test] [--fix_res] [--keep_res]
               [--not_rand_crop] [--shift SHIFT] [--scale SCALE]
               [--rotate ROTATE] [--flip FLIP] [--no_color_aug]
               [--aug_rot AUG_ROT] [--aug_ddd AUG_DDD] [--rect_mask]
               [--kitti_split KITTI_SPLIT] [--mse_loss] [--reg_loss REG_LOSS]
               [--hm_weight HM_WEIGHT] [--off_weight OFF_WEIGHT]
               [--wh_weight WH_WEIGHT] [--hp_weight HP_WEIGHT]
               [--hm_hp_weight HM_HP_WEIGHT] [--dep_weight DEP_WEIGHT]
               [--dim_weight DIM_WEIGHT] [--rot_weight ROT_WEIGHT]
               [--peak_thresh PEAK_THRESH] [--norm_wh] [--dense_wh]
               [--cat_spec_wh] [--not_reg_offset] [--agnostic_ex]
               [--scores_thresh SCORES_THRESH] [--center_thresh CENTER_THRESH]
               [--aggr_weight AGGR_WEIGHT] [--dense_hp] [--not_hm_hp]
               [--not_reg_hp_offset] [--not_reg_bbox] [--eval_oracle_hm]
               [--eval_oracle_wh] [--eval_oracle_offset] [--eval_oracle_kps]
               [--eval_oracle_hmhp] [--eval_oracle_hp_offset]
               [--eval_oracle_dep]
               task
main.py: error: the following arguments are required: task

**Comment by xingyizhou** [comment]
I have no idea why it happens. Please check your python version or change [this line](https://github.com/xingyizhou/CenterNet/blob/master/src/lib/opts.py#L13) to `    self.parser.add_argument('--task', default='ctdet',` and add `--task ctdet ' when running `main.py`.

**Comment by 201814476285951** [comment]
Thanks  a lot.
